demo/SFSW2_project_descriptions.R

```diff
@@ -89,7 +89,7 @@ project_paths <- list(
 
 #------ Base names or full names of input files
 fnames_in <- list(
-  fmaster = "SWRuns_InputMaster_YOURPROJECT_v11.csv",
+  fmaster = "SWRuns_InputMaster_Test_v11.csv",
```

Comment:
Should remain the dummy file here in the demo/ folder.

 fmaster = "SWRuns_InputMaster_YOURPROJECT_v11.csv",